vcpkg/get_started/get-started-msbuild.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Install and use packages with MSBuild in Visual Studio
+description: Tutorial guides the user through the process of installing and using packages with MSBuild and Visual Studio.
+author: data-queue
+ms.author: danshaw2
+ms.date: 10/03/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+---
+
+# Tutorial: Install and use packages with MSBuild in Visual Studio
+
+This guide sets up a simple C++ project using the library `zlib` with the MSBuild project build system.
+
+## Prerequisites:
+- Visual Studio 2015 Update 3 or later with the C++ Desktop Development workload
+- Windows 7 or newer
+- Git
+
+## Install `vcpkg`
+
+In a command prompt, run:
+```
+git clone https://github.com/microsoft/vcpkg
+cd vcpkg && .\bootstrap.bat
+.\vcpkg.exe integrate install
+```
+Outputs:
+```
+All MSBuild C++ projects can now #include any installed libraries. Linking will be handled automatically. Installing new libraries will make them instantly available.
+```
+
+The `bootstrap.bat` script fetches a copy of `vcpkg.exe` that we will use to install dependencies. The `integrate install` command sets the user wide vcpkg instance in `%APPDATA%\Local\vcpkg` so MSBuild will be able to find it.
+
+> [!NOTE]
+> The C++ Desktop Development workload for Visual Studio 17.6 and later comes with a copy of vcpkg, which is intended for experimental use. While you will be able to use this directly in the Visual Studio Developer Command Prompt, we recommend users to obtain a seperate copy of vcpkg instead.
+
+## Create a new project in Visual Studio
+
+Create a new console application project, **File > New Project > Console Application (C++ for Windows)**. This prints "Hello World" by default.
+
+## Add a vcpkg manifest to your project
+
+The [manifest](./manifest.md) will contain the set of C++ dependencies from vcpkg. In `Solution Explorer - Folder View`, right click to  **Add > New File > Type in "vcpkg.json"** to create a new manifest in the root of your project
+
+```json
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
+  "name": "my-application",
+  "builtin-baseline": "78ba9711d30c64a6b40462c72f356c681e2255f3",
+  "version": "0.15.2",
+  "dependencies": [
+    "zlib"
+   ]
+}
+```
+
+The builtin-baseline sets the global minimum version information for your manifest. While we can set it to any git commit hash in the vcpkg project, we recommend setting it to the latest commit. In the terminal where you acquired vcpkg, you can run `git rev-parse HEAD` or simply copy the latest commit from `git log`. For more information, see the [reference](../reference/vcpkg-json.md) on `vcpkg.json`. 
```

Comment:
```suggestion
The builtin-baseline sets the global minimum version information for your manifest. The baseline can be any commit hash in the vcpkg's repository history. We recommend setting the baseline to the latest commit. In the terminal where you acquired vcpkg, you can run `git rev-parse HEAD` or simply copy the latest commit from `git log`. For more information, see the [reference](../reference/vcpkg-json.md) on `vcpkg.json`. 
```

The last sentence should ask users to run `vcpkg x-update-baseline` instead of manually finding out the latest commit.